main.css

```diff
@@ -119,3 +124,30 @@ td.center {
 	font-style: italic;
 	font-size: 0.9em;
 }
+
+
+.toc {
+	background-color: lightgrey;
+	width: 200px;
+}
+
+.jump-button {
+	position: fixed;
+	display: flex;
+	bottom: 10px;
+	right: 10px;
+	width: 40px;
+	height: 40px;
+	background-color: lightblue;
+
+	justify-content: center;
+	align-items: center;
+
+	border-radius: 100%;
+}
+
+.jump-button > a {
+	font-size: xx-large;
+	text-decoration: none;
+	color: black;
```

Comment:
You should be able to center this fairly easily; if I have some time, I'll poke around and try.

Reply:
Centered it; tested on a bunch of different viewport sizes and seems to always work. Just added a bottom margin of 5px.